tf/start.sh

```diff
@@ -17,22 +18,43 @@ fi
 
 game_num=$(cat $GAMEFILE)
 game_num=$((game_num + GAMES))
-echo "Starting with training.$game_num.gz as last game in window"
+file="training.${game_num}.gz"
+
+echo "Starting with '$file' as last game in window"
+
+train() {
+  unbuffer ./train.py --cfg=$1 --output=$2 2>&1 > "$ROOT/logs/$(date +%Y%m%d-%H%M%S).log"
```

Comment:
Using redirect instead of tee means that training is now silent?

Reply:
Indeed. Could change it back though? But I think one should just `tail -f` the log files.

Reply:
I think you want >> instead of >, if you aren't going to switch back to tee.
I don't have a strong preference, but I think I liked being able to see the progress while the shell script was running without having to work out what todays log date is.

Reply:
I'll change back to `tee`